src/web-components/comments/css/annotation-item.style.ts

```diff
@@ -125,6 +125,25 @@ export const annotationItemStyle = css`
     width: 100%;
   }
 
+  .comments__hide-complete-annotation {
+    min-height: 0;
+    opacity: 0;
+    animation: hide-annotation 10s linear;
```

Comment:
10s?

Reply:
It's the time that the "resolve annotation" message takes to disappear